### PR TITLE
Enhance config documentation with examples

### PR DESCRIPTION
**Pull request by anslpa**

### CODE REVIEW

**Review by cbellot000 — APPROVED** [review]

File: src/ansys/dpf/core/config.py [hunk 1]
```diff
@@ -34,6 +34,24 @@ class Config:
...
+    --------
+    Create an operator configuration, instantiate the operator and
+    run it with the configuration.
+
+    >>> from ansys.dpf import core as dpf
+    >>> config_add = dpf.Config("add")
+    >>> config_add.set_work_by_index_option(True)
+    >>> op = dpf.operators.math.add(config=config_add)
+
+    Modify the copy of an operator's configuration and set it back.
```
Comment (JennaPaikowsky) [inline src/ansys/dpf/core/config.py:47]: By set it back, do you mean reset it? Or set it to something else?

File: src/ansys/dpf/core/dpf_operator.py [hunk 2]
```diff
@@ -474,12 +474,25 @@ def config(self):
         """Copy of the operator's current configuration.
 
         You can modify the copy of the configuration and then use ``operator.config = new_config``
-        or create an operator with the new configuration as a parameter.
+        or instantiate an operator with the new configuration as a parameter.
+
+        Refer to a specific operator's documentation to know about its options.
```
Comment (JennaPaikowsky) [inline src/ansys/dpf/core/dpf_operator.py:479]:
```suggestion
        For information on an operator's options, see the documentation for that operator.
```

**Review by JennaPaikowsky — COMMENTED** [review]
Just added a couple of things.

**Review by PProfizi — APPROVED** [review]

File: src/ansys/dpf/core/config.py (hunk 1 above)
Comment (anslpa) [reply src/ansys/dpf/core/config.py:47]: @JennaPaikowsky it means "replace the current operator's config by the modified copy of it"; would you have any suggestion for that ?

**Pull request merged**